owncloudApp/src/main/java/com/owncloud/android/ui/activity/FolderPickerActivity.java

```diff
@@ -509,4 +530,8 @@ private void refreshList(boolean ignoreETag) {
             }
         }
     }
+
+    public enum ACTION_BUTTON {
+        MOVE, COPY
```

Comment:
Consider adding all the potential uses, not only move and copy. By the way, could be a good idea to create a toString() function to retrieve the actual string id directly

Reply:
Done